jupyterhub/templates/proxy/autohttps/rbac.yaml

```diff
@@ -1,6 +1,17 @@
 {{- $HTTPS := (and .Values.proxy.https.hosts .Values.proxy.https.enabled) }}
 {{- $autoHTTPS := (and $HTTPS (eq .Values.proxy.https.type "letsencrypt")) }}
-{{- if (and $autoHTTPS .Values.rbac.enabled) -}}
+{{- if $autoHTTPS -}}
+{{- if .Values.proxy.traefik.serviceAccount.create }}
```

Comment:
The `-` to the right and left of the `{{` and ` }}` are related to "whitespace chomping".

A rule of thumb to avoid rendering surplus new lines etc we've tried to follow is:
1. Always whitespace chomp left
2. Never whitespace chomp right, _except for_ when it's before any previous content has been rendered in the template, then always whitespace chomp right as well.

For this line, we should also whitespace chomp right to follow that rule. It is not going to make things break etc, but it makes the output contain less surplus new lines when rendering it.